jobs/build/set_client_latest/set-latest.sh

```diff
@@ -18,13 +18,30 @@ TMPDIR=${WORKSPACE}/tools
 mkdir -p "${TMPDIR}"
 cd ${TMPDIR}
 
```

Comment:
This message is for line 10 (just adding here because I can't comment to line 10 where you don't make a change):
Could you also change the `curl` command line to include the `--fail` option so that it will fail the script if the response status code is not 2xx?
After adding that, the `| grep -q "404 Not Found"` and  line 12-15 can also be removed.